### PR TITLE
Added the algorithm to compute the time period of a simple pendulum

### PR DESCRIPTION
**Pull request by pluto-tofu**
### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [ ] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

**Comment by pluto-tofu** [comment]
@tianyizheng02 Could you please take a look at this pull request, I will make any necessary changes that you suggest.

File: physics/time_period_simple_pendulum.py
Comment (tianyizheng02) [inline physics/time_period_simple_pendulum.py:1]: Please rename the file and the function to `period_of_pendulum` to keep the names simple.

File: physics/time_period_simple_pendulum.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    >>> time_period_simple_pendulum(-12)
+    Traceback (most recent call last):
+        ...
+    ValueError: The length should be non-negative
+    >>> time_period_simple_pendulum(0)
+    0.0
+    """
+    if length < 0:
+        raise ValueError("The length should be non-negative")
+    return (2 * 3.14159) * (length / g) ** 0.5
```
Comment (tianyizheng02) [inline physics/time_period_simple_pendulum.py:53]: For more accurate calculations, please import pi from a library such as `math`

**Review by tianyizheng02 — CHANGES_REQUESTED** [review]

**Comment by pluto-tofu** [comment]
@tianyizheng02 Sorry, I was a little late in making the requested changes but I committed the changes yesterday and it's showing outdated as of now. Could you please look into it.

File: physics/period_of_pendulum.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    >>> period_of_pendulum(-12)
+    Traceback (most recent call last):
+        ...
+    ValueError: The length should be non-negative
+    >>> period_of_pendulum(0)
+    0.0
+    """
+    if length < 0:
+        raise ValueError("The length should be non-negative")
+    return (2 * pi) * (length / g) ** 0.5
```
Comment (tianyizheng02) [inline physics/period_of_pendulum.py:46]:
```suggestion
    return 2 * pi * (length / g) ** 0.5
```
Minor nitpick: the parentheses around the `2 * pi` are unnecessary

**Review by tianyizheng02 — APPROVED** [review]

**Pull request merged**